dockerfiles/testsuite/start_test.sh

```diff
@@ -165,42 +163,36 @@ for test in ${tests_to_run[@]}; do
 done
 
 log "### Waiting for tests to finish ###"
-wait
 
-failed=""
-for test in ${tests_to_run[@]} ; do
-	RET=$(cat $results/$test.retval)
-	if [ "$RET" == "0" ] ; then
-		log "### Vochain test $test finished correctly! ###"
-	else
-		log "### Vochain test $test failed! ###"
-		log "### Post run logs ###"
-		$COMPOSE_CMD logs --tail 1000
-		failed="$test"
-		break
-	fi
-done
+wait_jobs() {
+	while [ -n "$(jobs)" ]; do
```

Comment:
just to learn, where the variable `jobs` is defined?

Reply:
i can't recommend you to learn bash :see_no_evil:
but here we go: `jobs` is not a var but a command